local/blockchain.go

```diff
@@ -990,7 +990,7 @@ func (*localNetwork) createBlockchains(
 			defaultPoll,
 		)
 		if err != nil {
-			return fmt.Errorf("failure issuing create blockchain: %w", err)
+			return fmt.Errorf("P-Wallet Tx Error %s %w, blockchainID %s", "Issue Create Blockchain Tx", err, blockchainTxs[i].ID().String())
```

Comment:
can you use IssueCreateBlockchainTx instead of "Issue Create Blockchain Tx"? for consistency with the other err msgs
also, probably all IDs in the other err msgs will need a String() call as in this one (eg nodeID.String() etc)